tree_unique_args/src/schedule.egg

```diff
@@ -3,11 +3,12 @@
 (run-schedule
   (repeat 6
     (saturate always-run)
-    (saturate error-checking) ; In the future, this will be "debug mode" only
+    (saturate error-checking) ; In the future, this will be "release mode" only
```

Comment:
What does this mean? Is it because error-checking is too slow for rustc's debug mode? I would think we need to do error checking if we are in *eggcc*'s debug mode. A small clarification here would be helpful.